core/src/main/java/com/sequenceiq/cloudbreak/service/customconfigs/CustomConfigurationsService.java

```diff
@@ -194,6 +199,7 @@ public CustomConfigurations deleteByCrn(String crn) {
 
     public CustomConfigurations deleteByName(String name, String accountId) {
         CustomConfigurations customConfigurationsByName = getByName(name, accountId);
+        LOGGER.info("Deleting custom configurations: {}", customConfigurationsByName);
```

Comment:
do we need to show the whole object here?

Reply:
Can there be any sensitive data here?

Reply:
Updated this